rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.88.0"
+channel = "stable"
```

Comment:
I think this should always be stable, the `rust-version` field is the thing that actually pins the code and behavior.